README.md

```diff
@@ -10,34 +10,29 @@ The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.
 
 ## Cylc to run the pipeline
 
-Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
+Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. The default pipeline is built to run on Brown's Oscar HPC and each task is submitted as it's own batch job. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
 
 ### Running the Cylc pipeline on Oscar
 
 1. - [ ] Connect to Oscar from VS Code
     * [use this guide](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/remote-ide)
 
-2. Move to a compute node
-   - [ ] `interact -n 20 -t 24:00:00 -m 32g`
-    * this will start a compute session for 1 day with 32 GB memory and 20 cores
-    * see [here](https://docs.ccv.brown.edu/oscar/submitting-jobs/interact) for more options
```

Comment:
So Cylc can manage this kind of thing on its own?

Reply:
It is covered by a combination of VS Code connecting to a compute node and then each task being submitted to slurm with memory and cores.